rasqc/checkers/volume_accounting.py

```diff
@@ -0,0 +1,75 @@
+"""Module for simulation volume accounting checks."""
+
+from ..base_checker import RasqcChecker
+from ..registry import register_check
+from ..rasmodel import RasModel
+from ..result import RasqcResult, ResultStatus
+
+from rashdf import RasPlanHdf
+
+from pathlib import Path
+from typing import List
+
+VOL_ERR_PER_TOL = 2
```

Comment:
Can we spell this out more fully? Wasn't sure what `PER_TOL` meant at first glance